### ISSUE TITLE
Unable to resolve "../../../core/src/errors" in analytics-react-native-plugin-firebase

### ISSUE THREAD

**Issue by enagorny** [segmentio/analytics-react-native#815]
After updating to a latest versions getting error in firebase plugin @segment/analytics-react-native-plugin-firebase
Unable to resolve "../../../core/src/errors" from "node_modules/@segment/analytics-react-native-plugin-firebase/src/FirebasePlugin.tsx"

- `analytics-react-native` version: "@segment/analytics-react-native": "^2.13.5",
- Integrations versions (if used): "@segment/analytics-react-native-plugin-firebase": "^0.3.8",
- React Native version: 0.71.6
- iOS or Android or both? both

**Steps to reproduce**
Install latest versions and try to use firebase plugin

**Expected behavior**
should work

**Actual behavior**
Import error

**Comment by enagorny** [comment]
Managed to solve it for me with patch-package 

```diff
diff --git a/node_modules/@segment/analytics-react-native-plugin-firebase/src/FirebasePlugin.tsx b/node_modules/@segment/analytics-react-native-plugin-firebase/src/FirebasePlugin.tsx
index 522312b..b1caed9 100644
--- a/node_modules/@segment/analytics-react-native-plugin-firebase/src/FirebasePlugin.tsx
+++ b/node_modules/@segment/analytics-react-native-plugin-firebase/src/FirebasePlugin.tsx
@@ -4,13 +4,13 @@ import {
   PluginType,
   ScreenEventType,
   SegmentError,
+  ErrorType,
   TrackEventType,
 } from '@segment/analytics-react-native';
 import screen from './methods/screen';
 import track from './methods/track';
 import reset from './methods/reset';
 import firebaseAnalytics from '@react-native-firebase/analytics';
-import { ErrorType } from '../../../core/src/errors';
 
 export class FirebasePlugin extends DestinationPlugin {
   type = PluginType.destination;
```

**Comment by alanjcharles** [comment]
Hi @enagorny thanks for letting us know. I just made a fix in #816, I'll let you know when it's available.

**Comment by alanjcharles** [comment]
@enagorny available in [0.3.8](https://www.npmjs.com/package/@segment/analytics-react-native-plugin-firebase). Thanks for bringing this to our attention!

**Comment by rezkiy37** [comment]
> @enagorny available in [0.3.8](https://www.npmjs.com/package/@segment/analytics-react-native-plugin-firebase). Thanks for bringing this to our attention!

@alanjcharles Sorry, but I don't see this fix 👀
<img width="1039" alt="Screenshot 2023-04-27 at 18 59 44" src="https://user-images.githubusercontent.com/57314631/234935958-6582f7ba-2fc3-41da-bf71-cebeb6937ed5.png">

**Comment by joseocabarcas** [comment]
I have this problem. 0.3.8 don't fix

**Comment by alanjcharles** [comment]
@rezkiy37 @joseocabarcas @enagorny sorry folks- fixed in [0.3.9](https://www.npmjs.com/package/@segment/analytics-react-native-plugin-firebase)

**Comment by rezkiy37** [comment]
> @rezkiy37 @joseocabarcas @enagorny sorry folks- fixed in [0.3.9](https://www.npmjs.com/package/@segment/analytics-react-native-plugin-firebase)

Thank you!
